akka-remote/src/main/scala/akka/remote/artery/AeronSink.scala

```diff
@@ -184,6 +197,14 @@ class AeronSink(
         failStage(cause)
       }
 
+      private def onPublicationClosed(): Unit = {
+        offerTaskInProgress = false
+        val cause = new PublicationClosedException(s"Aeron Publication to [${channel}] was closed.")
+        flightRecorder.loFreq(AeronSink_PublicationClosed, channelMetadata)
```

Comment:
your call, depends how serious you think the condition is. `alert` will take a snapshot of the log, while loFreq will not.